e2-studio-star-rx72n-firmware/tests/test_rx_pid.c

```
@@ -717,29 +717,29 @@ void test_pid_compute_integral_accumulation(void)
   TEST_ASSERT_EQUAL(k_rx_ok, rx_pid_init(&pid, &config));
 
-  float output   = 0.0f;
-  float setpoint = 100.0f;
-  float measured = 90.0f;
-  float error    = 10.0f;
-  float dt       = 0.01f;
+  float output   = 0.0F;
+  float setpoint = 100.0F;
+  float measured = 90.0F;
+  float error    = 10.0F;
+  float dt       = 0.01F;
```

Clang (14.0.6)

[warning] variable name 'dt' is too short, expected at least 3 characters

(readability-identifier-length)